src/oop_ext/interface/_tests/test_interface.py

```diff
@@ -342,6 +342,24 @@ def m2(self):
     _InterfM1(m1)  # Not raise BadImplementationError
 
 
+def testStubsFromInstances():
+    """Check that creating interface stubs (``foo = IFoo(foo)``) work as expected"""
+
+    class IFoo(Interface):
+        def foo(self):
+            ...
+
+    class Foo:
+        def foo(self):
+            return 10
+
+    stub = IFoo(Foo())
```

Comment:
Weird to create instance of interface.

Reply:
I agree, in retrospect it would better to have a specific function for that in `interface`:

```python
stub = interface.make_stub(Foo(), IFoo)
```

This was only done like this so PyDev would autocomplete `stub`. 😛